wasmtime/_types.py

```diff
@@ -425,3 +429,6 @@ def type(self):
     def __del__(self):
         if self.__owner__ is None:
             dll.wasm_exporttype_delete(self.__ptr__)
+
+
+ExternalType = typing.Union[FuncType, TableType, MemoryType, GlobalType]
```

Comment:
Could this use similar naming to the other union? Something like `IntoExternType`?

Reply:
- [ ] I'll look into this.

I'll look at `wasmtime.h` and read up on this Rusty `Into` thing.

Personally (strong opinion, weakly held), I'm very happy having things named the same across different modules. I'm used to naming things super close to what's used in the domain, and if I end up with the same names across modules, I'll just use `from fruit import Banana as Banana_` or `import fruit` then use `fruit.Banana` in the code. I'm already struggling to cope with the names of things in this package not matching those in `wasm.h` or the wasmtime API or the WebAssembly spec as it is: for example, I think `Instance` should be called `ModuleInstance`, since there are _instances_ of several other things in the runtime environment. So starting to add `Into` at the start might be a move too far. :-)

Reply:
Er well in any case I think the naming here and the actual instances themselves should be the same. For example `External` and `ExternalType`